module3/index.md

```diff
@@ -16,5 +16,6 @@ In Module 3, students will begin to dive into the skills and mindsets necessary
 
 ## Curriculum
 
-* [Lessons](./lessons)
-* [Projects](./projects)
+### Week 1
```

Comment:
Makes sense to me not to have a prep for this lesson because it's day 1 of Mod 2.